coralproject/talk
# [CORL-2503] Fix seen comments replies race condition

## What does this PR do?

Fixes the issue where some comments in the stream were not reliably being marked as seen.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indexes added.

## How do I test this PR?

- delete the existing `seenComments` collections you have
- view stream with a logged in user
- view replies and subscriptions, etc
- ensure that when you refresh the page and navigate, all seen comments are properly marked as seen
 
## How do we deploy this PR?

No special considerations for deployment.
